src/components/MerchantsPage/gam/GiftAMealPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './styles.module.scss';
+import CampaignListItem from './CampaignListItem'
+
+import gam_hero from '../images/gam_hero.png'
+import gam_1 from '../images/gam_1.svg'
+import gam_2 from '../images/gam_2.svg'
+import gam_3 from '../images/gam_3.svg'
+import gam_4 from '../images/gam_4.svg'
+
+interface Props {
+    menuOpen: boolean;
+}
+
+const GiftAMealPage = (props: Props) => {
```

Comment:
Did you run `yarn format`? This file seems a bit more indented than the rest of our files

Reply:
Oops! I did not!